### PR TITLE
[NCL-3896] Cancel BuildConfigSet endpoint

### PR DESCRIPTION
**Pull request by michalovjan**
@matejonnet Could you please advise with MDC in the new endpoint

### CODE REVIEW

**Comment by michalovjan** [comment]
retest this please

File: rest/src/main/java/org/jboss/pnc/rest/endpoint/RunningBuildRecordEndpoint.java [hunk 1]
```diff
@@ -140,4 +147,32 @@ public Response getAllForBCSetRecord(
...
+    @ApiResponses(value = {
+            @ApiResponse(code = SUCCESS_CODE, message = SEARCH_DESCRIPTION),
+            @ApiResponse(code = NOT_FOUND_CODE, message = NOT_FOUND_DESCRIPTION),
+            @ApiResponse(code = SERVER_ERROR_CODE, message = SERVER_ERROR_DESCRIPTION)
+    })
+    @POST
+    @Path("/build-config-set-records/{id}/cancel")
+    public Response cancelAllBuildsInGroup(
+            @ApiParam(value = "Build Configuration Set id", required = true) @PathParam("id") Integer bcSetRecordId) {
+        // TODO MDC
```
Comment (janinko) [inline rest/src/main/java/org/jboss/pnc/rest/endpoint/RunningBuildRecordEndpoint.java]: This todo should be deleted.

**Comment by michalovjan** [comment]
retest this please

**Comment by michalovjan** [comment]
retest this please

**Pull request merged**